src/logic/image-processor.ts

```diff
@@ -54,6 +56,13 @@ export async function createImageProcessor({
             }
           }
 
+          const deploymentTimestamp = deploymentTimestamps.get(result.entity)
+          if (deploymentTimestamp) {
+            const approximateDurationPerAvatar = duration / avatars.length
+            const durationSeconds = (approximateDurationPerAvatar - deploymentTimestamp) / 1000
```

Comment:
Can we review this calculation? I may be wrong but, let's suppose:
- This takes 20 seconds to convert 2 avatars
- Average is 10 seconds
- We are doing (10 - 1755874793000) / 1000

I think we should do (for each avatar conversion):
- Date.now() - 1755874793000 == time